README.md

```diff
@@ -22,6 +22,16 @@ Cryptographic operations in CyberChef should not be relied upon to provide secur
 
 [A live demo can be found here][1] - have fun!
 
+## Running in Docker
```

Comment:
This section might be better put lower down in the page.  People will likely want to read what the project does before learning how to run it.  Maybe around the contributing section?

Reply:
My thought process here was that running it locally goes hand in hand with the live demo but I can put it where ever you want it to go.

Reply:
I moved the section to where it was suggested.